vsts/pipelines/nightly.yml

```diff
@@ -48,8 +48,8 @@ jobs:
     displayName: 'Set variables'
   - template: _buildParallel.yml
 
-- job: Job_IntegrationTests
-  displayName: Test Integration
+- job: Job_PythonIntegrationTests
```

Comment:
Basically you can have a template for a `job` for integration tests itself (related to what you mentioned in scrum meeting today)...job templates can be passed the parameters too

Reply:
yes, similarly I want have template for build image and test image etc etc